docs/source/user_guide/model_understanding.ipynb

```diff
@@ -565,6 +565,84 @@
     "                                        include_shap_values=True, output_format=\"dataframe\")\n",
     "report"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Force Plots\n",
+    "Force plots can be generated to predict single or multiple rows for binary, multiclass and regression problem types.  Here's an example of predicting a single row on a binary classification dataset.  The force plots show the predictive power of each of the features in making the negative (\"Class: 0\") prediction and the positive (\"Class: 1\") prediction. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shap\n",
+    "\n",
+    "from evalml.demos import load_breast_cancer\n",
+    "from evalml.model_understanding.force_plots import graph_force_plot\n",
+    "from evalml.pipelines import BinaryClassificationPipeline\n",
+    "\n",
+    "X, y = load_breast_cancer()\n",
+    "\n",
+    "\n",
+    "pipeline = BinaryClassificationPipeline(component_graph = ['Simple Imputer', 'Random Forest Classifier'])\n",
+    "pipeline.fit(X, y)\n",
```

Comment:
You don't need to fit a pipeline here. We fit one at the top of the model understanding guide. Please delete.

https://evalml.alteryx.com/en/v0.24.1/user_guide/model_understanding.html